packages/utils/src/file.ts

```diff
@@ -0,0 +1,29 @@
+import fs from "fs";
+
+export type ArchiveType =
+  | "state"
+  | "signedBlock"
+  | "block"
+  | "attestation"
+  | "signedAggregatedAndProof"
+  | "syncCommittee"
+  | "contributionAndProof";
+
+/**
+ * Persist bad items to archive dir, for example invalid state, attestations etc.
+ */
+export function archiveItem(type: ArchiveType, bytes: Uint8Array, suffix = ""): string {
+  const now = new Date();
+  // yyyy-MM-dd
+  const date = now.toISOString().split("T")[0];
+  const parentDir = process.env.LODESTAR_ARCHIVE_DIR;
```

Comment:
This is problematic, utils must be able to run in the browser and this is beacon specific code. I would prefer to have this code in packages/lodestar only unless there's a strong reason to put it here